### ISSUE TITLE
Missing X closing icon on "Register for applicant" popup modal

### ISSUE THREAD

**Issue by kribac12** [NoroffFEU/agency.noroff.dev#557]
- Missing X on add image pop-up on register for applicant
- Look at the difference between popup modal on "Register for companies" vs. "Register for Applicants". Both should have "X" to close modal, but only the one on "Register for companies" has.
- The code used for Register for companies should already have what is needed to fix this. 

- Issue raised before, but was closed, possibly due to believing it was fixed because the close icon was visible on the "Register for companies" modal. 

<img width="495" alt="Screenshot 2024-01-09 at 14 15 54" src="https://github.com/NoroffFEU/agency.noroff.dev/assets/89196018/3834a577-c75c-4286-b4ce-39eedbc9c1a8">
<img width="678" alt="Screenshot 2024-01-09 at 14 16 10" src="https://github.com/NoroffFEU/agency.noroff.dev/assets/89196018/3ce70a26-5577-4fba-be97-1c320879ba29">

**Comment by mhaarseth** [comment]
@NoroffFEU/korketrekkeren Initial testing shows this error manifests differently in different browsers. In Firefox a clickable black square shows up and in Chrome nothing shows up. Could anybody test this in safari and tell me how it looks there?

(And then there's the issue of the double "add image" button in the register for companies modal, but that's not in the scope of this issue).

**Comment by kimrm** [comment]
In safari there's no closing icon.

![Image](https://github.com/NoroffFEU/agency.noroff.dev/assets/20448821/62c8163f-6716-4bec-844c-07be2087ee4f)

**Comment by mhaarseth** [comment]
Thanks!

**Comment by mhaarseth** [comment]
@NoroffFEU/korketrekkeren Easy fix, but keep getting conflicts on pull request. I deleted the branch on github, and will delete my local branch before trying again. Have done the prettier fix as talked about on teams, but it didn't seem to do anything, so I'll just start completely from scratch. 

If I keep encountering conflicts on the pull request, is that the QAs or my job to fix?

**Comment by mhaarseth** [comment]
I will, if the problem persists :) Giving it another shot now.

**Comment by mhaarseth** [comment]
Hey, @kimrm! I still get the conflict in the pull request. Could you have a look at it and report back?

**Comment by kimrm** [comment]
You can click on the link in the PR and check:


![Image](https://github.com/NoroffFEU/agency.noroff.dev/assets/20448821/4977e586-0d3e-42f0-86c1-84eac5f97ce4)

Looks like your file is formatted with prettier and the remote one is not.

**Comment by mhaarseth** [comment]
I'll check it out, but I'll need to read up on the documentation for conflict resolving on github. Will get back to you.

**Comment by mhaarseth** [comment]
After chatting and researching for some time, @kimrm and I realised this has to do with me running "git pull" on my local projects doesn't actually pull changes from the remote main, so I had been working on the wrong file all along. I fixed it by running "git pull origin main". Both commands should work the same, but in this case they didn't. Oh, the joys of developing! I will be closing the pull request and start again, working on the right file this time.

**Comment by kribac12** [comment]
@mhaarseth  Great that you discovered what was wrong. I will move this to "Doing" then, and when ready for QA you move it back and tag Siw with the PR number.

**Comment by mhaarseth** [comment]
I am starting at this  - again - from scratch now, thursday 11.01. Saw some discussion about the codebase as a whole on discord, so I will read through that first, to see if anything applies to this.

**Comment by mhaarseth** [comment]
@NoroffFEU/korketrekkeren After a lot of trouble getting Vite to run so I could continue working on this, it seems the issue has been fixed by someone else already. Could one of you check to see if the X to close is there now, in the add image modal, when opened from "register for Applicant"? I would normally just trust my own eyes, but after all the trouble I'm worried I did something to make my local changes effect what I get up  when opening the page from the main branch.
